pkg/core/blockchain.go

```diff
@@ -1316,6 +1317,25 @@ func (bc *Blockchain) verifyTx(t *transaction.Transaction, block *block.Block) e
 	if t.ValidUntilBlock <= height || t.ValidUntilBlock > height+transaction.MaxValidUntilBlockIncrement {
 		return errors.Errorf("transaction has expired. ValidUntilBlock = %d, current height = %d", t.ValidUntilBlock, height)
 	}
+
+	account := bc.GetAccountState(t.Sender)
+	if account == nil {
+		return errors.Errorf("account not found: %v", address.Uint160ToString(t.Sender))
+	}
+	balance := account.GetBalanceValues()[UtilityTokenID()]
+	fee := bc.SystemFee(t) + bc.NetworkFee(t)
+	if balance < fee {
+		return errors.Errorf("insufficient funds: balanse is %v, fee is %v", balance, fee)
+	}
+	for _, tx := range bc.memPool.GetVerifiedTransactions() {
+		if tx.Tx.Sender == t.Sender && tx.Tx != t {
+			fee += tx.Fee + bc.SystemFee(tx.Tx)
+		}
+	}
+	if balance < fee {
+		return errors.Errorf("insufficient funds: balanse is %v, total sender fee is %v", balance, fee)
+	}
```

Comment:
I think we can completely drop this part until we have sys fee and net fee fields in `Transaction` (#831).